recipes/xtensor/run_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
The library is header-only, but running the tests has the same issues as libdynd... We need a c++14 compiler.